1-mandatory/2-people-i-know.js

```diff
@@ -378,14 +378,22 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+thirtyFiveOrOlder = friends.filter((friend) => {
```

Comment:
Nitpick: For a single statement you don't need a full bracket syntax. The same things can be made simpler and more readable without it.
```
friends.filter(friend => friend.age >= 35)
```

Reply:
The prettier editor setting keeps putting the brackets back around (friend) every time I hit save.